src/Symfony/Component/Notifier/Bridge/MessageBird/Tests/MessageBirdTransportFactoryTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\MessageBird\Tests;
+
+use Symfony\Component\Notifier\Bridge\MessageBird\MessageBirdTransportFactory;
+use Symfony\Component\Notifier\Test\TransportFactoryTestCase;
+use Symfony\Component\Notifier\Transport\TransportFactoryInterface;
+
+final class MessageBirdTransportFactoryTest extends TransportFactoryTestCase
+{
+    /**
+     * @return MessageBirdTransportFactory
+     */
+    public function createFactory(): TransportFactoryInterface
+    {
+        return new MessageBirdTransportFactory();
+    }
+
+    public function createProvider(): iterable
+    {
+        yield [
+            'messagebird://host.test?from=0611223344',
+            'messagebird://:authToken@host.test?from=0611223344',
```

Comment:
Shall we remove the `:` ?

Reply:
I think it is not possible

```
1) Symfony\Component\Notifier\Bridge\MessageBird\Tests\MessageBirdTransportFactoryTest::testCreate with data set #0 ('messagebird://host.test?from=...223344', 'messagebird://authToken@host....223344')
Symfony\Component\Notifier\Exception\IncompleteDsnException: Invalid "messagebird://authToken@host.test?from=0611223344" notifier DSN: Password is not set.

```

Reply:
In this case using `$this->getUser()` instead of `$this->getPassword()` in the TransportFactory should do the trick